opencage_geocoder.go

```diff
@@ -115,8 +134,13 @@ func (g *OpenCageGeocoder) ReverseGeocode(p *Point) (string, error) {
 // Return sthe first address in the passed in byte array.
 func (g *OpenCageGeocoder) extractAddressFromResponse(data []byte) string {
 	res := &opencageGeocodeResponse{}
-	json.Unmarshal(data, &res)
+	if err :=  json.Unmarshal(data, &res) ; err != nil {
+        panic(err)
+    }
```

Comment:
It looks like a `panic` was added here?  You mentioned in your Pull Request that you were removing a `panic`?

For what it's worth, the [GoogleGeocoder equivalent](https://github.com/zingbats/golang-geo/blob/dont_panic/google_geocoder.go#L134-L147) is returning `(string, error)` instead of just a `string`.  I think that's a good model to follow here, as well.